manifests/image-references

```diff
@@ -2,11 +2,11 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
-  - name: cluster-group-upgrades-operator-precache
+  - name: telco-5g-ran4-topology-aware-lifecycle-operator-precache-rhel8
     from:
       kind: DockerImage
       name: quay.io/openshift-kni/cluster-group-upgrades-operator-precache:4.10.0
-  - name: manager
+  - name: openshift4-topology-aware-lifecycle-manager-rhel8-operator
```

Comment:
this should also just be repository name: `topology-aware-lifecycle-manager-rhel8-operator`